xbmc/addons/Scraper.cpp

```diff
@@ -785,6 +792,16 @@ void DetailsFromFileItem<CArtist>(const CFileItem& item, CArtist& artist)
   int nFanart = item.GetProperty("artist.fanarts").asInteger();
   artist.fanart.m_xml = ParseFanart(item, nFanart, "artist.fanart");
   artist.fanart.Unpack();
+
+  std::map<std::string, std::string> artwork;
+  std::string logo = FromString(item, "artist.clearlogo.url");
+  if (!logo.empty())
+    artwork.insert(std::pair<std::string, std::string>("clearlogo", logo));
+  std::string banner = FromString(item, "artist.banner.url");
+  if (!banner.empty())
+    artwork.insert(std::pair<std::string, std::string>("banner", banner));
+  if (!artwork.empty())
```

Comment:
Minor, but you could re-use the same var for every value if you only want to transfer it into the map.

```
std::string value = FromString(...
if (!value.empty())
  // transfer value into map
value = FromString(...
if (!value.empty())
  // transfer value into map

```